pom.xml

```diff
@@ -323,6 +324,42 @@
       </properties>
     </profile>
 
+    <!-- arm soft float profile -->
+    <profile>
+      <id>armsf</id>
```

Comment:
I think this should be linux-armsf, because there could be other OSes as well. Maybe even use the conventions from the OS maven-plugin?

Reply:
I'm not aware of too many cross-compilers we'll be supporting, so the likelihood of needing to differentiate is slim and we can certainly make it more specific once we decide we need more.  For example in the event we decide to support Windows ARM, Solaris ARM, AIX ARM, macOS ARM, we'll have reasoning to differentiate then.  For now the goal is to match the `PROFILE` with the `CMAKE_TOOLCHAIN_FILE` which should be easy to locate.  I recommed we keep them simple until we need more `TOOLCHAINS`.

In regards to the plugin naming convention matching our toolset, I too find this to be ideal long term but for now I'd like to avoid `PROFILE=windows_x86_64_64` or `PROFILE=mingw-w64_x86_64_64` or `PEOFILE=linux_aarch64_64` when the shorter form is adaquate.